code/modules/mob/living/living_defines.dm

```diff
@@ -146,3 +146,12 @@
 
 	/// This is the cooldown on suffering additional effects for when we exhaust all stamina
 	COOLDOWN_DECLARE(last_stamina_exhaustion)
+
+	///The world.time of when this mob was last lying down
+	var/last_rested = 0
+	///The world.time of when this mob became unconscious
+	var/last_unconscious = 0
+	///The world.time of when this mob entered a stasis bag
+	var/time_entered_stasis = 0
+	///The world.time of when this mob entered a cryo tube
+	var/time_entered_cryo = 0
```

Comment:
these seem kinda pointless and mob var is :/

Unconsious can also be tracked with status effects being added instead

Reply:
Not vital or anything I just thought players would like knowing how long they've really been incapacitated or left unattended in stasis/cryo. Problem with using the status effect is what if it ends earlier than the timer argument? Then it gets inaccurate.